src/docs/product/releases/index.mdx

```diff
@@ -8,70 +8,88 @@ redirect_from:
 
-Using one of Sentry's repository integrations (such as GitHub, GitLab, Bitbucket, and so forth) is the easiest way to connect your commit metadata to Sentry. For a list of available integrations, go to Organization Settings > Integrations.
+Notifying Sentry of a release and providing commit metadata can be done most easily using one of our repository integrations. We also support a [manual process](/product/releases/manual-setup/without-repo) if you either don’t want Sentry to connect to your repository or you’re using an unsupported repository provider or VCS.
 
-<Note>
+<Alert level="warning" title="Required Permissions">
 
-You need to be an Owner or Manager of your Sentry organization to set up or configure an integration. Read more about [roles in Sentry](/product/accounts/membership/).
+To set up or configure an integration, you need to be an Owner or Manager of your Sentry organization. Learn more about [roles in Sentry](/product/accounts/membership/).
 
-</Note>
+If you’re linking a GitHub repository, you must have Admin or Owner permissions for the repository, and Sentry must be an authorized GitHub app in your [GitHub account settings](https://github.com/settings/applications).
+
+</Alert>
+
+1. Install the applicable integration. The available integrations are listed under your **Organization Settings > Integrations**.
+
+2. Click the "Configure" button next to your instance.
 
-![](releases-repo-integrations.png)
+   ![](releases-repo-integrations.png)
 
-Once you are in Organization Settings > Integrations and have installed the integration, click the 'Configure' button next to your instance.
+3. In the **Repositories** panel, click "Add Repository" to add any repositories from which you'd like to track commits. The integration will then send Sentry metadata (such as authors and files changed) about each commit pushed to those repositories.
 
-![](releases-repo-add.png)
+  ![](releases-repo-add.png)
 
-In the 'Repositories' panel, click 'Add Repository', and add any repositories you'd like to track commits from. The integration will then send Sentry metadata (such as authors and files changed) about each commit pushed to those repositories.
+  <Note>
 
-If you’re linking a GitHub repository, ensure you have Admin or Owner permissions on the repository, and that Sentry is an authorized GitHub app in your [GitHub account settings](https://github.com/settings/applications).
+  If you’re having trouble adding the integration, [disconnect](https://sentry.io/account/settings/identities/) then [reconnect](https://sentry.io/account/settings/social/associate/github/) your GitHub identity.
 
-If you’re still having trouble adding it, you can try to [disconnect](https://sentry.io/account/settings/identities/) and then [reconnect](https://sentry.io/account/settings/social/associate/github/) your GitHub identity.
+  </Note>
 
-### Create Release and Associate Commits {#create-release}
+### Create the Release and Associate Commits
```

Comment:
Please consider leaving these anchors in. This allows you to change the wording of the headings without breaking any links across the page, as shown in this PR: Most of the changed files are just updates to anchors, which can be avoided with stable anchors.

Reply:
I've fixed all the broken links already, so we don't need to preserve these anchors. They are also now generated by the new heading (so this anchor is now #create-the-release-and-associate-commits).